tools/c7n_gcp/c7n_gcp/resources/kms.py

```diff
@@ -0,0 +1,131 @@
+# Copyright 2019 Capital One Services, LLC
```

Comment:
this seems suspect as well

Reply:
Mmm, this still seems in line with current general practice in this repo - I see other entries, e.g. in c7n_kube, azure. If you believe this is incorrect and a blocker - could you remove/update this line yourself to what you think is a correct statement, thanks. 
CC: @jondavidi

Reply:
@kapilt this is inline with the general practices that seem to be followed in the project here and elsewhere (e.g. gcp_azure). I don't think this should be a blocking PR issue while it's in discussion and hope you agree. If there's still concerns about it, I would recommend following up with @jonathana to discuss this, as I mindful that it has been a topic that has come up before

Reply:
we've been fixing this in new contributions but we haven't gone back to fix previous contributions. feel free to read the contributor agreement.

Reply:
the correct statement is a copyright mark and year from the business entity or individual if not contributing on behalf a business entity that wrote the new file or substantial portion of the file in question. employment and contracted work directly between individual or corporate entity is another valid basis for changing copyright ownership although subject to contract terms of those respective relationships. Which happens not to be the case here, as capitalone is not the contracting entity to your business. of course check your contract and lawyer if you wish and IANAL, but having a bunch of factually untrue copyright statements is not something we wish to propagate or condone in the project. Other projects have normalized such copyright statements declarations (i.e. kubernetes, etc) across all contributors and authors to a project name authors and contributors, but we haven't taken any formal directions towards such, as its most befitting projects in a neutral foundation.

Reply:
@kapilt is this PR good to move forward, per the conversation we had about copyrights not being a blocking issue the other week?

Reply:
Agreed per conversation copyright wouldn’t be a blocker then noting that we’re still awaiting clarification for this contrib on its status, however there are other items.

Reply:
@kapilt, do you mean there is something else that needs resolving in this PR? From my perspective the comments were addressed, please point out the things that need my attention if I'm wrong.

Reply:
@kapilt, did you have a chance to take a look at the latest changes? Please let me know if there is anything that needs changing.